### ISSUE TITLE
Editor: is disabled and data is clear after minimizing the dialog

### ISSUE THREAD

**Issue by geekdiv** [primefaces/primefaces#3206]
I have `p:editor` (also try `pe:ckEditor`) inside my dialog. when I minimize and restore the dialog again, the data in editor is gone and editor is disabled. 

I have tried both primefaces editor and primefaces ckEditor but facing same issue on both.

## 1) Environment

- primefaces-6.1

- It does not work on Primeface showcase as well

- primefaces-extensions-6.1.1

- jsf-2.2

## 2) Expected behavior

After restore dialog editor data not clear and editor should not disable.

## 3) Actual behavior

After restore dialog editor data is clear and editor is disable. Instead of editor data display 'true' like below screenshot (Using Firefox)
![image](https://i.stack.imgur.com/SMIqc.png)

## 4) Steps to reproduce

Open Dialog add some data in editor and minimize dialog then again maximize that dialog.

## 5) Sample XHTML
```
 <p:dialog minimizable="true" modal="false" id="newDlg" widgetVar="dlgNew">
        	<p:editor value="test"/>
 </p:dialog>
```
```
<p:dialog closeOnEscape="true" responsive="true" widgetVar="dlg" id="dlg1" resizable="false" draggable="true" closable="true" dynamic="true" modal="false"  width="97%" minimizable="true" appendTo="@(body)" header="Dialog" onShow="PF('dlg').initPosition()">
   <pe:ckEditor toolbar="#{config['ckEditor.toolBar']}" id="body" widgetVar="editor" readonly="false" width="100%" maxlength="50000" value="#{myBean.textVal}" />
</p:dialog>
```

**Comment by tandraschko** [comment]
NOTE: p:editor is already deprecated and will be removed someday - so this will likely not be fixed. However, i would create a ticket in PFE to fix that for CKEditor.

**Comment by geekdiv** [comment]
OK, Please create ticket in PFE please share me that thread link so I followup regularly and check more update regrading that issue.
Thank You  @tandraschko :)

**Comment by tandraschko** [comment]
Sorry, it would be great if you could create the ticket there ;)

**Comment by melloware** [comment]
@tandraschko new ticket is in PFE.  Safe to close this ticket.

**Comment by melloware** [comment]
Fixed in 14.0.0
